src/app/share/error.tsx

```
@@ -0,0 +1,28 @@
+"use client"; // Error boundaries must be Client Components
+
+import { useEffect } from "react";
+import Header from "../components/header";
+import { useRouter } from "next/navigation";
+
+export default function Error({
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)